docs/repositories/issues.md

```diff
@@ -61,7 +61,7 @@ Use the options in the cogwheel menu of each issue to:
     See [how to restore ignored issues](#restoring-ignored-issues).
 
     !!! tip
-        Organization owners can [configure who is allowed to ignore issues](../organizations/roles-and-permissions-for-synced-organizations.md#configure-ignore-issues).
+        Organization owners can [configure who is allowed to ignore issues](../organizations/roles-and-permissions-for-synced-organizations.md#change-analysis-configuration).
```

Comment:
Perhaps we can add an analogous "tip" card on the page [Code patterns](https://docs.codacy.com/repositories-configure/code-patterns/), at least temporarily to give more visibility to this new feature. :thinking:

Reply:
Looks a bit out of scope, can you create an issue for this?

Reply:
I would prefer to make this change in the same scope of this pull request, as it is a direct consequence of allowing organization owners to configure who can make changes to code patterns. I can make the change myself, though.